firebase_admin/_auth_providers.py

```diff
@@ -253,6 +278,16 @@ def _make_request(self, method, path, **kwargs):
             raise _auth_utils.handle_auth_backend_error(error)
 
 
+def _validate_oidc_provider_id(provider_id):
+    if not isinstance(provider_id, str):
+        raise ValueError(
+            'Invalid OIDC provider ID: {0}. Provider ID must be a non-empty string.'.format(
```

Comment:
This will not check for empty strings though, will it?

Reply:
The immediate next check (`if not provider_id.startswith('oidc.'):`) will cover empty strings.